CONTRIBUTING.md

```diff
@@ -0,0 +1,114 @@
+For a new issue on the [Github issues](https://github.com/Tuhura-Tech/Wiki/issues) issues page, press 'New issue'   
+From there decide which issue you would like to raise:
+- Docs Request - this is for if you have information that you would like to be documented on the Tūhura Tech wiki, or you have a request for something you would like us to look into and document
+- Report a security vulnerability - if you have seen a security vulnerability, let us know privately here
+
+### Documentation request
+
+#### Title
+Try and keep the title for what you want to be request short and simple.  
+A good title is no more than a few words, for example "Python Introduction" or "Database backend" are good titles.
+
+#### Short description
+This is to expand your request into a full sentence or two. Try and keep this simple without going into detail.  
+A good example of a short description is "Please write a guide on how to use markdown"
+
+#### Detailed/additional information 
+This is optional.  
+If you have information you would like to be cut down and added to the wiki add it here. 
+
+Or, if you have screenshots to help explain what you would like to be added paste them here.  
+The easiest way to add an image is to `right click > copy image` on most platforms like Google Chrome or Discord, and then using `ctrl + v` in the issue to paste it as a markdown link
+
+## Directly writing documentation
+It's awesome you want to help us document and create resources for people to learn from! To get started you will need to set a few things up
+
+### Setting up
+To contribute you first need to hold a copy of our wiki on your device to edit it. First off, create a [fork of the repository](https://github.com/Tuhura-Tech/Wiki/fork)
+
+The simplest way to clone [the repository](https://github.com/Tuhura-Tech/Wiki) is to use [Github Desktop](https://desktop.github.com).   
+
+Github Desktop is available for [Windows](https://central.github.com/deployments/desktop/desktop/latest/win32) and [macOS](https://central.github.com/deployments/desktop/desktop/latest/darwin)  
+
+Open the installer and let it run. Then "Clone a repository from the Internet", which can also be reached from `File > Clone repository...` or `ctrl + shift + o`. Go to the third tab and paste in the link to your cloned branch and choose an empty local path, then press 'Clone'.
+
+From here you can open it in your file explorer or preferred code editor via the buttons or manually in those programs
+
+### Writing in markdown 
+[markdownguide.org](https://www.markdownguide.org) has a wonderful [cheat sheet](https://www.markdownguide.org/cheat-sheet/) for writing in markdown. All of the documentation is written in markdown so knowing this is crucial. If you would like to see examples of how to format your documentation go to the documentation we have already written and look at the raw markdown. The most commonly used syntax are # to indicate titles and subtitles so be sure to get familiar with that.
```

Comment:
We might also want to include the docs for mermaid as we use that for diagrams https://emersonbottero.github.io/vitepress-plugin-mermaid/